akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -381,6 +408,11 @@ sealed abstract class ByteString extends IndexedSeq[Byte] with IndexedSeqOptimiz
   def asByteBuffers: immutable.Iterable[ByteBuffer]
 
   /**
+   * Indicates whether this ByteString can produce a wrapped ByteBuffer via [[asByteBuffer]].
```

Comment:
These can be tricky, but I think the right syntax is `[[ByteString#asByteBuffer]]`.